ansible_rulebook/cli.py

```diff
@@ -105,6 +105,19 @@ def get_parser() -> argparse.ArgumentParser:
         "default to yes for wss connection.",
         default="yes",
     )
+    parser.add_argument(
+        "--websocket-access-token",
+        help="Token used to autheticate the websocket connection.",
+    )
+    parser.add_argument(
+        "--websocket-refresh-token",
+        help="Token used to renew a websocket access token.",
+    )
+    parser.add_argument(
+        "--websocket-token-address",
+        "--websocket-token-url",
+        help="Url to renew websocket access token.",
```

Comment:
@bzwei can we use environment variables here, would provide an easy way to test in the current setup https://github.com/ansible/ansible-rulebook/blob/8743f53a1a7bf3eb24dfbc4cbe03c1cd1fad66da/ansible_rulebook/cli.py#L125